ls1intum/Artemis
# show number of active users

### Checklist

### Motivation and Context

This change shows the number of active users in the application and is displayed in the application metrics section.


Issue #1330 

### Description


### Steps for Testing


1. Log in to Artemis
2. Navigate to Server Administration
3. Then go to Metrics
4. See the 'Active Users' field in the 'System' section.

### Screenshots

![Screenshot 2020-05-03 at 17 07 59](https://user-images.githubusercontent.com/33379863/80917683-fc731a00-8d60-11ea-95cb-df40e89a4f13.png)
